generator/src/Main.elm

```diff
@@ -57,11 +57,16 @@ dropIndexFromLast path =
         |> List.reverse
 
 
+chopForwardSlashes : String -> String
+chopForwardSlashes =
+    String.split "/" >> List.filter ((/=) "") >> String.join "/"
```

Comment:
FYI, I want to use the `Pages.Internal.String` module here, but didn't know how to share that code with the CLI.

Reply:
Ah yes. I'll keep that in mind for the future. It's pretty minor so this seems good for now to me 👍